zanata-war/src/main/java/org/zanata/rest/service/SourceDocResourceService.java

```diff
@@ -421,7 +421,7 @@ private HLocale validateTargetLocale(LocaleId locale, String projectSlug,
         } catch (ZanataServiceException e) {
             log.warn(
                     "Exception validating target locale {} in proj {} iter {}",
-                    e, locale, projectSlug, iterationSlug);
+                    locale, projectSlug, iterationSlug);
```

Comment:
This is simply wrong. You can't mix template with exception stacktrace. This was treating the exception as first variable and log its toString as locale.